src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java

```diff
@@ -415,7 +415,7 @@ public List<OffenderIdentifier> getOffenderIdentifiers(final Long bookingId, @Nu
                 .toList();
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER"})
+    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
```

Comment:
This is needed to allow us to remove the SYSTEM_USER role from our endpoints and client creds